src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -145,8 +145,9 @@ def generate_FT_S_PAT_default_config(self, filename):
     open(os.path.join('testcases', 'testdata', 'device_a.json')))
 
     # load FSS
-    fss_record_0 = json.load(
+    fss_data = json.load(
```

Comment:
I guess the TT&C information doesn't need to be used in this test case, so it's OK to discard it. (No change needed.)